dictionary.json

```diff
@@ -103,6 +109,13 @@
       "header_type": "dictionary",
       "expected": {"a": [1,{}], "b": [true, {"foo": 9}], "c": [3, {}]}
     },
+    {
+        "name": "missing value with params dictionary, irregular serialisation",
+        "raw": ["a=1, b;foo=9, c=3"],
```

Comment:
This isn't allowed by either the ABNF or the algorithm. Parameters are part of the member value, but the member value is omitted in this serialisation.

Reply:
Logged as [#1007](https://github.com/httpwg/http-extensions/issues/1007) in case the test is correct, and the spec has a bug.